Automattic/wp-calypso
# Plugin Browser: show active/inactive state on plugin card

#### Changes proposed in this Pull Request

* Adds active status to the plugin card in the `PluginBrowser`

#### Testing instructions
* Navigate to one of the paths provided below
* Check if it show active/inactive state or the amount of sites is shown depending on the selected url
* It should render the list in all the urls, check for any wsod

Please test that **active/inactive** state is shown in the following urls:
```
/plugins/:site
/plugins/:site?s=<query>
/plugins/:category/:site
```

Please test that **active/inactive** state is not shown in the following urls:
```
/plugins (no site)
/plugins/?s=<query>
/plugins/:category (no site)
```

#### Screenshots
| Mobile | Desktop |
|-------|------|
| ![Screen Shot 2022-02-23 at 3 56 51 PM](https://user-images.githubusercontent.com/1035546/155407137-7ae15ce8-2825-4e8b-aa84-4cac352a5c6f.png)| ![Screen Shot 2022-02-23 at 3 56 22 PM](https://user-images.githubusercontent.com/1035546/155407069-a1c3a3d0-b46f-42df-a115-1bf8b569d45d.png)|


Related to #58539
